### ISSUE TITLE
Links broken

### ISSUE THREAD

**Issue by tpaixao** [razorx89/digital-hand-atlas-downloader#1]
Hi, it seems the Ipilab db is broken/doesn't work. Is there a new url, or is it gone forever? I would really like to get my hands on the data (especially the clinical data, like readings from the 2 radiologists, tanner index, etc...)

**Comment by razorx89** [comment]
Unfortunately, the website is broken since February this year, I think. I reported it to the maintainers (broken website, php server is spilling code, user/password leak for the database, publicly accesssible database, ...), but since there has been no fix for over half a year now I think the website won't be fixed anymore. You can still try to contact them in order to request the original DICOM dataset. Unluckily, the DICOM data is good to extract the images, but the meta informations are incomplete and had to be completed by the online table content.

**Comment by valsevalse** [comment]
Check this out, there is a download link
https://ipilab.usc.edu/computer-aided-bone-age-assessment-of-children-using-a-digital-hand-atlas-2/

**Comment by tpaixao** [comment]
Thanks, but that link is for the images only,. I was interested in the
other data (bond age readings from doctors, etc...)

Thanks though.

**Comment by razorx89** [comment]
Part of the data is present in the DICOM header. Unfortunately, the data seems to come from different sources, half of it contains all meta data, the other half misses some values (if I remember correctly, the readings were missing).

**Comment by valsevalse** [comment]
My understanding is that the name of the folder of each JPG image has the age in it, is that right?

**Comment by razorx89** [comment]
If I remember correctly it is the chronological age (on a yearly basis). This has nothing to do with radiologist readings.

**Comment by tpaixao** [comment]
Exactly, the DICOM headers do contain the birth date and the acquisition 
date, which allows you to calculate the chronological age (up to the day 
- which is more precise than simply the year class). However, there were 
also bone age readings from 2 radiologists (and other data, like BMI, 
height and so on) which are not present in the DICOM headers.

One Note: there are about 6 images where this data seem inaccurate (eg, 
one image is clearly from a baby while the chronological age calculated 
from the DICOM headers gives something like 14 years)

Having the bone-age as read by doctors is important if you want for 
example to calculate the inter-observer variability and also because the 
Greulich-Pyle scale is a bit outdated and needs to be corrected to 
modern populations.

Sven Koitka wrote:

**Comment by jonathancolledge** [comment]
@tpaixao Did you ever find the bone age readings?
